defaults/main.yml

```diff
@@ -1,6 +1,15 @@
 # true if IPv6 is needed
 network_ipv6_enable: false              # sshd + ssh
 
+# true if sshd should be started and enabled
+ssh_server_enabled: false               # sshd
```

Comment:
Could you set this to true, please?

The goal of this role is that it should work out-of-the-box, but without restarting the role would not fulfil this.